tests/ert3/config/test_ert3_config.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+import pydantic
+from ert3.config import _ensemble_config
```

Comment:
Import of private member? Also, I suggest we strive to keep the imports at `import ert3` and nothing more..,.

Reply:
In a test I would argue that it is fine to import a private function (and also to test a private function) As for the imports, in tests I am in favor of importing just the relevant functions to keep the scope minimal. In production code I am fine with sticking to `import ert3`

Reply:
I'm not going to be absolute on this, but in general I think that importing private members in tests is often a symptom of some other entity having too much responsibility. In this case, we have handed the `run` method the responsibility of loading the configuration... Which it probably should not have over time!

But, as you have now made the `load_ensemble_config` public, we have a supported way of creating these objects and I think we should use that one in the tests as well...

Reply:
What do you mean by keeping the scope minimal? In particular, I find it difficult to see that we'll be able to enforce a different coding standard in tests and implementation 🤷 Also, it defies the purpose of having the tests be a natural consumer of the implementation as intended to be used...

Reply:
It highlights that the point of this test is just to test `_ensemble_config` in my opinion. I disagree that the tests should just be a natural consumer of the implementation. Take this config as an example. It is a layered structured by nature, and in this case basically everything is required. That means the natural interaction is through `load_ensemble_config`, but the layers are individually unit testable. Just testing everything though the `load_ensemble_config` would make the tests include a lot of boiler plate. If a lot was possible to default, the story would be different in my opinion. Having it like this makes it easier to distinguish the responsibility of the different layers. Unless I am misunderstanding completely 😂